fog/ingest/server/src/controller.rs

```diff
@@ -975,6 +978,11 @@ where
         self.recovery_db.get_missed_block_ranges()
     }
 
+    pub fn get_last_report_published_block_index(&self) -> Result<u64, ()> {
+        // Should return be Result<u64, ()> ?
```

Comment:
Should just return `u64` if it can't error.

Reply:
Sarah suggested an rpc_internal_error down in the ingest_service.rs file changes below...I think that eventually bubbles up to here?

Reply:
`rpc_internal_error` is in the GRPC service implementation, which is what calls this method. However, since this is a simple getter, there is no scenario it which it can fail.

Reply:
uneeded since using get-status now